<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>737</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>737</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string> </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #36393F;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-size: 12px;
	color: #fff;
	background: #4A77F2;
	text-decoration: none;
	border-radius: 10px;
	transition: .5s;
}
QPushButton:hover{
	color: #4A77F2;
	background: #FFF;
}
QPushButton:pressed{
	background: #CCCCCC;
  	box-shadow: 0 3px rgb(33,147,90) inset;
	
}</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="text">
     <string>Фирсов Кирилл все права защищены©</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_city">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	display: flex;
	position: absolute;
	background-color: #fff;
	height: 40px;
	border-radius: 10px;
	padding: 10px;

	border: none;
	outline: none;
	font-size: 16px;
	color: #36393F;
	transition: .4s;
	line-height: 40px;
	width: 0;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Город</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="text">
     <string>Все хорошо</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-size: 12px;
	color: #fff;
	background: #4A77F2;
	text-decoration: none;
	border-radius: 10px;
	transition: .5s;
}
QPushButton:hover{
	color: #4A77F2;
	background: #FFF;
}
QPushButton:pressed{
	background: #CCCCCC;
  	box-shadow: 0 3px rgb(33,147,90) inset;
	
}</string>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-size: 12px;
	color: #fff;
	background: #4A77F2;
	text-decoration: none;
	border-radius: 10px;
	transition: .5s;
}
QPushButton:hover{
	color: #fff;
	background: red;
}
QPushButton:pressed{
	background: #CCCCCC;
  	box-shadow: 0 3px rgb(33,147,90) inset;
	
}</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLabel" name="city_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="status_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Users/Кирилл/Desktop/temp_good.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="temp_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="city_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temp_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="status_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Users/Кирилл/Desktop/temp_not_good.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="city_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temp_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="status_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Users/Кирилл/Desktop/temp_super_not_good.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="forecast_btn_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-size: 12px;
	color: #fff;
	background: #4A77F2;
	text-decoration: none;
	border-radius: 10px;
	transition: .5s;
}
QPushButton:hover{
	color: #4A77F2;
	background: #FFF;
}
QPushButton:pressed{
	background: #CCCCCC;
  	box-shadow: 0 3px rgb(33,147,90) inset;
	
}</string>
    </property>
    <property name="text">
     <string>Прогноз погоды</string>
    </property>
   </widget>
   <widget class="QPushButton" name="average_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-size: 12px;
	color: #fff;
	background: #4A77F2;
	text-decoration: none;
	border-radius: 10px;
	transition: .5s;
}
QPushButton:hover{
	color: #4A77F2;
	background: #FFF;
}
QPushButton:pressed{
	background: #CCCCCC;
  	box-shadow: 0 3px rgb(33,147,90) inset;
	
}</string>
    </property>
    <property name="text">
     <string>Средняя температура</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_city_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	display: flex;
	position: absolute;
	background-color: #fff;
	height: 40px;
	border-radius: 10px;
	padding: 10px;

	border: none;
	outline: none;
	font-size: 16px;
	color: #36393F;
	transition: .4s;
	line-height: 40px;
	width: 0;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Город</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>1</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>800</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="notification_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-size: 12px;
	color: #fff;
	background: #4A77F2;
	text-decoration: none;
	border-radius: 10px;
	transition: .5s;
}
QPushButton:hover{
	color: #4A77F2;
	background: #FFF;
}
QPushButton:pressed{
	background: #CCCCCC;
  	box-shadow: 0 3px rgb(33,147,90) inset;
	
}</string>
    </property>
    <property name="text">
     <string>Создать напоминание о погоде</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_city_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	display: flex;
	position: absolute;
	background-color: #fff;
	height: 40px;
	border-radius: 10px;
	padding: 10px;

	border: none;
	outline: none;
	font-size: 16px;
	color: #36393F;
	transition: .4s;
	line-height: 40px;
	width: 0;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Город</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Дата/время</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>194</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff; border: none; outline: none; border-radius: 10px;</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="actionmain">
   <property name="text">
    <string>main</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
